frontend/src/Metamaps/Cable.js

```diff
@@ -1,5 +1,7 @@
 /* global $, ActionCable */
 
+import { indexOf } from 'lodash'
```

Comment:
because `indexOf` was already being used on line 114 in the code, so maybe it would have broke if it actually hit that code. I just don't know how it knew to import off lodash